src/org/zaproxy/zap/extension/importLogFiles/ExtensionImportLogFiles.java

```diff
@@ -1,3 +1,22 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright The ZAP development team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */

```

Comment:
I'm not sure why this file was missing a license header :man_shrugging: Could we get you to use this one: https://github.com/zaproxy/zap-admin/blob/master/docs/headers/license.java (replacing `$YEAR` with `2014`? (https://github.com/zaproxy/zap-extensions/commits/e772e050351022bb23aee50c741661f6c0a152e5/src/org/zaproxy/zap/extension/importLogFiles/ExtensionImportLogFiles.java)

Reply:
Sure!